Gemfile

```diff
@@ -30,6 +30,7 @@ gem 'slim-rails'                                                   # slim templa
 gem "sprockets", "~> 3.7.2"                                        # sprockets is a rack-based asset packaging system that concatenates and serves javascript, scss, etc
 gem 'sucker_punch', '~> 2.0'                                       # asynchronous processing library
 gem 'uglifier', '>= 1.3.0'                                         # compressor for javascript assets
+gem 'nokogiri', '>= 1.8.3'
```

Comment:
What would you think about using `~> 1.8.3` instead so this locks to the correct minor version?

Reply:
I agree with @pachonk.  @arun057 - Canyou make that tweak?

Reply:
🤦‍♂️ Sorry about that.

Reply:
@phallstrom Done :slightly_smiling_face: